.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ experimental:
 templates:
   job_template: &job_template
     docker:
-      - image: datadog/datadog-agent-runner-circle:go11612
```

Comment:
This also affects all other components, are we sure it's a good idea to change it so late in the release process?

Reply:
I agree that we don't want to include the revert of the Go upgrade from 1.16.7 to 1.16.12 (https://github.com/DataDog/datadog-agent/pull/10404) as part of this PR, unless it's directly related to the NPM regression.

1.16.12 includes security fixes that we want to keep in 7.34